js/main.js

```diff
@@ -25,5 +25,81 @@ $form.addEventListener('submit', function (event) {
 
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
 
+  $entryformDiv.className = 'hidden';
+  $entriesDiv.className = 'entries';
+  data.view = 'entries';
   $form.reset();
+  window.location.reload();
```

Comment:
😱 😱 😱 😱 

By doing this, you are reloading the entire page, which is extremely resource-intensive. Rather than reload the page, you should be creating a DOM tree for the new entry, and add it to the DOM yourself at the top of your list. That way the entry is visible without refreshing the page.